ArcadiaTeamsBot/Cards/Cards.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace ArcadiaTeamsBot.Cards
+{
+    using System.Collections.Generic;
+
+    using Microsoft.Bot.Schema;
+
+    public class Cards
+    {
+        public static HeroCard GetChoiceCard()
```

Comment:
These cards are related to a single dialog, they don't have to be in separate class.

Reply:
Replaced